dev-client/src/screens/ScreenScaffold.tsx

```diff
@@ -63,6 +65,7 @@ export const ScreenScaffold = ({
       />
       <Column backgroundColor="primary.contrast" flex={1}>
         <View onLayout={onLayout}>{PropsAppBar}</View>
+        {isFlagEnabled('FF_offline') && <SyncButton />}
```

Comment:
question: Would it make sense to a Flag Restricted component, like we do for permissions? i.e.
```
<RestrictByFlag flag="FF_offline" component={<SyncButton />} />
```

or
```
<RestrictByFlag flag="FF_offline"><SyncButton /></RestrictByFlag>
```

Reply:
I think this would be a good idea.

Reply:
Thanks, will do! Any preferences on if it should go within the SyncButton, or if it should be the caller's responsibility to put it around the SyncButton, or both? (I don't expect there to be many callers so shouldn't matter too much)